### PR TITLE
Update prep & install docs for IBM

### PR DESCRIPTION
**Pull request by Rachael-Graham**
Prep:
- Move Managed Istio add-on section out of this page, bc this page is only for setting up the cluster & getting kube config
- Update name of docs pages in the IKS doc set and remove region-specific text (regions are being deprecated; zones now only)
- Update cli command to require min worker number, worker machine type

Setup:
- Move Managed Istio add-on section into the IBM Cloud Public section (can't be used w/ private)
- Add UI steps & CLI steps for installing add-on
- Add screenshot for UI steps

### CODE REVIEW

File: content/docs/setup/kubernetes/install/platform/ibm/index.md [hunk 1]
```diff
@@ -11,16 +11,66 @@ aliases:
...
+
+    {{< text bash >}}
+    $ kubectl get svc -n default
+    {{< /text >}}
+
+    {{< text bash >}}
+    $ kubectl get pods -n default
+    {{< /text >}}
+
+### Helm
```
Comment (frankbu) [inline content/docs/setup/kubernetes/install/platform/ibm/index.md]:
```suggestion
### Manual Istio install
```

File: content/docs/setup/kubernetes/install/platform/ibm/index.md [hunk 2]
```diff
@@ -11,16 +11,66 @@ aliases:
...
 
 Follow [these instructions](/docs/setup/kubernetes/install/helm/) to install and run the current release version of Istio in [IBM Cloud Public](https://www.ibm.com/cloud/) using Helm and the IBM Cloud Kubernetes Service (IKS).
```
Comment (frankbu) [inline content/docs/setup/kubernetes/install/platform/ibm/index.md]:
```suggestion
Follow [these instructions](/docs/setup/kubernetes/) to install and run the current release version of Istio in [IBM Cloud Public](https://www.ibm.com/cloud/) using the IBM Cloud Kubernetes Service (IKS).
```

File: content/docs/setup/kubernetes/install/platform/ibm/index.md [hunk 3]
```diff
@@ -11,16 +11,66 @@ aliases:
 Follow this flow to install and configure an Istio mesh in IBM Cloud.
 
 You can use the [managed Istio add-on for IBM Cloud Kubernetes Service](#managed-istio-add-on)
-in IBM Cloud Public, use Helm to install Istio in [IBM Cloud Public](#ibm-cloud-public),
+in IBM Cloud Public, use [Helm to install Istio in IBM Cloud Public](#helm),
```
Comment (frankbu) [inline content/docs/setup/kubernetes/install/platform/ibm/index.md]: Why only helm? I think this should say something like:
```
in IBM Cloud Public, [install Istio manually](#manual-istio-install) in IBM Cloud Public
```

**Comment by Rachael-Graham** [comment]
@frankbu TY for your review! For some reason it marked you as not having signed the CLA though when I accepted your suggestions?

**Pull request merged**